web/tests/user_events.test.js

```diff
@@ -27,9 +27,6 @@ mock_esm("../src/activity_ui", {
 mock_esm("../src/compose_state", {
     update_email() {},
 });
-mock_esm("../src/narrow_state", {
-    update_email() {},
-});
```

Comment:
Removing this didn't cause any issues, and having it here was causing test failures when other functions in `narrow_state` were being called

Reply:
Generally we're very happy to mock fewer things, so great.